up42/project.py

```diff
@@ -16,10 +16,6 @@ class Project:
     new workflows, query already existing workflows & jobs in the project and manage
     the project settings.
 
-    Create a new project on the
-    [**UP42 Console website**](https://sdk.up42.com/authentication/#get-your-project-credentials).
-
-    Use an existing project:
     ```python
     up42.authenticate(project_id="uz92-8uo0-4dc9-ab1d-06d7ec1a5321",
```

Comment:
Why not to drop this line as well? Also in __init__.py docstring

Reply:
Added the suggestion from Daria, pointing to the authentication guidelines.